spec/bump/commands/validate_spec.rb

```diff
@@ -3,17 +3,18 @@
 describe Bump::CLI::Commands::Validate do
   it "calls the server and exit successfully if success" do
     stub_bump_api_validate("validations/post_success.http")
+    allow(URI).to receive(:open).and_return(spy(read: "content"))
 
     expect {
       new_command.call(
         doc: "aaaa0000-bb11-cc22-dd33-eeeeee444444",
-        'doc-name': "Hello world",
+        "doc-name": "Hello world",
```

Comment:
I was surprised in codestyle commit e53d626, that this was not automatically handled by `standardrb`

Reply:
Yes, and I just saw that there are still some issues. I will run it another time and manually fix what needs to be fixed.